tests/e2e/commands/network.go

```diff
@@ -81,11 +81,30 @@ func StartNetworkWithVersion(version string) string {
 }
 
 /* #nosec G204 */
-func StopNetwork() {
+func StopNetwork(stopCmdFlags ...string) {
+	stopCmdFlasg := append([]string{
+		NetworkCmd,
+		"stop",
```

Comment:
I wonder if we should start using a constants for the command names, defined under cmd/ tree